### PR TITLE
Moving pkg/kubectl/util/*.go to staging

### PR DESCRIPTION
**Pull request by pswica**
-  Moves pkg/kubectl/util/*.go to staging.
- Publishes this code to the new kubectl repository: https://github.com/kubernetes/kubectl/pkg/util
- Code will be available for re-use after publishing.

/kind cleanup
/sig cli
/area kubectl
/priority important-longterm
/hold
/assign @seans3 

```release-note
NONE
```

### CODE REVIEW

File: hack/boilerplate/boilerplate.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def file_extension(filename):
 
 skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh',
                 "vendor", "test/e2e/generated/bindata.go", "hack/boilerplate/test",
-                "pkg/kubectl/generated/bindata.go"]
+                "staging/src/k8s.io/kubectl/pkg/generated/bindata.go"]
```
Comment (seans3) [inline hack/boilerplate/boilerplate.py]: This file should not have references to files within staging.

File: hack/generate-bindata.sh [hunk 2]
```diff
@@ -65,7 +65,7 @@ fi
 rm -f "${BINDATA_OUTPUT}.tmp"
 
 # These are files for runtime code
-BINDATA_OUTPUT="pkg/kubectl/generated/bindata.go"
+BINDATA_OUTPUT="staging/src/k8s.io/kubectl/pkg/generated/bindata.go"
```
Comment (seans3) [inline hack/generate-bindata.sh]: This file should not have references to files within staging.

**Review by seans3 — CHANGES_REQUESTED** [review]
Files outside of the `pkg/kubectl` tree should not reference into this tree. Conceptually, we are moving into an entire new repository `kubernetes/kubectl`. So any references will go bad once we move out of staging and into our own repository. We need to update these references so that they are under `staging/src/k8s.io/kubectl`.

**Comment by pswica** [comment]
/cc @nikhita /cc @apelisse 
PTAL
This is the last thing to move in pkg/kubectl/util except some sensitive generated code

**Comment by apelisse** [comment]
/close

I think this PR is obsolete, since a lot of things moved to staging already. Feel free to re-open if needed.

**Pull request closed without merging**